cmake/Install.cmake

```diff
@@ -26,23 +26,36 @@ if (UNIX)
   set(CPACK_BINARY_NSIS    OFF CACHE BOOL "Recommended OFF")
 
   if(APPLE)
-    set(CPACK_BINARY_IFW ON CACHE BOOL "Enable to build IFW package, which is the recommended method")
-    set(CPACK_BINARY_STGZ    OFF CACHE BOOL "Recommended OFF")
+    set(CPACK_BINARY_IFW   ON CACHE BOOL "Enable to build IFW package, which is the recommended method")
+    set(CPACK_BINARY_STGZ OFF CACHE BOOL "Recommended OFF")
 
     # Mac Specific options to turn off
     set(CPACK_BINARY_BUNDLE        OFF CACHE BOOL "Recommended OFF")
     set(CPACK_BINARY_DRAGNDROP     OFF CACHE BOOL "Recommended OFF")
     set(CPACK_BINARY_OSXX11        OFF CACHE BOOL "Recommended OFF")
-    set(CPACK_BINARY_PACKAGEMAKER  OFF CACHE BOOL "This was the legacy method on Apple, superseded by IFW.")
+    set(CPACK_BINARY_PACKAGEMAKER  OFF CACHE BOOL "This was the legacy method on Apple, superseded by IFW")
     set(CPACK_BINRARY_PRODUCTBUILD OFF CACHE BOOL "Recommended OFF")
 
   else()
-    set(CPACK_BINARY_IFW     OFF CACHE BOOL "This should be off")
-    set(CPACK_BINARY_STGZ ON CACHE BOOL "Enable to build a Linux sh installer script, which is the recommended method") # Uses STGZ currently (install .sh script CACHE BOOL)
+    # TODO: Make IFW recommended? Deprecate STGZ?
+    set(CPACK_BINARY_IFW   ON CACHE BOOL "Enable to build IFW package, which is the recommended method")
+    set(CPACK_BINARY_STGZ  ON CACHE BOOL "Enable to build a Linux sh installer script, which is the legacy method")
 
     # Unix (non Apple CACHE BOOL) specific option to turn off
     set(CPACK_BINARY_TZ  OFF CACHE BOOL "Recommended OFF")
   endif()
+
+  # TODO: the "FORCE" is temporary to avoid people having an existing build directory miss the fact that the recommended method changed
+  # TODO: remove after next release
+  if (UNIX AND NOT APPLE)
+    if(NOT CPACK_BINARY_IFW)
+      set(CPACK_BINARY_STGZ  OFF CACHE BOOL "This was the legacy method on Linux, superseded by IFW" FORCE)
+      set(CPACK_BINARY_IFW    ON CACHE BOOL "Enable to build IFW package, which is the recommend method" FORCE)
+      message("Switching from STGZ to IFW as the supported generator has changed on Linux")
+    endif()
+  endif()
+  # END TODO
```

Comment:
I don't usually like forcing things, but I understand the intent here.  I'm fine with this, or leaving it out.  We only have a few developers who ever build packages, so it's not like it's widespread with people building packages everywhere.